pyvisa/resources/resource.py

```diff
@@ -208,6 +208,9 @@ def resource_info(self) -> highlevel.ResourceInfo:
     #: VISA attributes require the resource to be opened in order to get accessed.
     #: Please have a look at the attributes definition for more details
 
+    #: The session of the Resource Manager used to open this session.
+    resource_manager_session: Attribute[int] = attributes.AttrVI_ATTR_RM_SESSION()
+
```

Comment:
Since the same information is accessible through `resource_manager.session` I think it is not worth exposing it here. Having it define allow to have a uniform behavior for `get_attribute` which is important but exposing it would just be confusing IMO.